RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release
+
+The kubernetes-base-addons repository is intended to support a minimal base set of Addons required to provide monitoring, logging, alerting, and backups for every version of Kubernetes actively supported by the upstream Kubernetes community.
+It is not intended to be tied to any specific version of Kubernetes installer (ie. Konvoy or Kommander), or Kubeaddons.
+Changes to Konvoy or Kommander to create a resource needed by an Addon should be avoided.
+
+- [Schedule](#schedule)
+- [Considerations](#considerations)
+  - [Kubernetes](#kubernetes)
+  - [Branches and Tags](#branches-and-tags)
+- [Process](#process)
+  - [Testing Release (Weekly, Thursday)](#testing-release-weekly-thursday)
+  - [Stable Release (Weekly, Wednesday)](#stable-release-weekly-wednesday)
+
+## Schedule
+
+Releases should be weekly on Wednesdays, or as needed to address CVEs.
+
+## Considerations
+
+### Kubernetes
+
+The Addons in this repo are the minimum base set of supported<sup>[1](#footnote1)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+
+Kubernetes support must be maintained for the latest general release of Kubernetes and the two prior minor releases.
+At the time of this writing, the latest Kubernetes release is 1.17.2.
+Support from this repo, at this time, should cover 1.15.x, 1.16.x, and 1.17.x.
+
+### Branches and Tags
+
+As much as possible, we will try to maintain a `master` branch that is compatible with all supported versions of Kubernetes.
+If there is a variation in the Kubernetes API which requires a _**breaking change**_ to the Addon, a _**branch**_ will be made for the prior Kubernetes versions.
+All future changes adopted into master will need to be back-ported to those branches.
+
+## Process
+
+### Testing Release (Weekly, Thursday)
+
+Each _**Thursday**_, this repository should be tagged for SOAK testing as follows:
+
+- Parse the PR logs for release notes and generate and commit a Changelog.md
```

Comment:
I do believe that we should ask the PR submitter to update a running Changelog.md, instead of doing that before the release.

Reply:
IMHO only if we find tooling that can automate it. we should try to avoid manual steps in this repo.

Reply:
I think @joejulian was suggesting using the same setup from PR https://github.com/mesosphere/konvoy/pull/1219 to automate all that

Reply:
I don't think the release tooling solves all the problems. The people running the release process most likely does not have the context on each change, and is probably not likely to produce accurate and informative release notes as compared to the folks that write the PR.

Reply:
No no, so what PR https://github.com/mesosphere/konvoy/pull/1219 does is add tooling that enables a way to grep all release-notes from PRs. Any original author who creates a PR still has to add release-notes because, yes exactly, they are the ones with the most context on the code changes.

Reply:
Right, the PR is the time at which we will need to enforce that the release notes are sufficiently detailed to remove the burden from a human.

Reply:
+1 on PR creator to also write the release notes

Reply:
Added text to make it clear that this should be using automated tooling.